experimental/addons/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,235 @@
+# opentelemetry-operator
+
+This is an addon template. Check how to build your own addon: https://kubevela.net/docs/platform-engineers/addon/intro
+
+## Install
+
+Add experimental registry
+```
+vela addon registry add experimental --type=helm --endpoint=https://addons.kubevela.net/experimental/
+```
+
+Enable this addon
+```
+vela addon enable opentelemetry-operator
+```
+
+```shell
+$ vela ls -A | grep opentelemetry
+vela-system     addon-opentelemetry-operator    ns-opentelemetry-operator               k8s-objects                                     running healthy
+vela-system     └─                              opentelemetry-operator                  helm                                            running healthy Fetch repository successfully, Create helm release
+```
+
+Disable this addon
+```
+vela addon disable opentelemetry-operator
+```
+
+## Use opentelemetry-operator
+
+### Opentelemetry Collector
+
+Once the opentelemetry-operator addon enabled successfully, you can deploy OpenTelemetry Collector in your Kubernetes cluster.
+
+The Collector can be deployed as one of four modes: `Deployment`, `DaemonSet`, `StatefulSet` and `Sidecar`. The default mode is `Deployment`.
+
+**Collector as sidecar**
+
+The following YAML creates a pod with a sidecar collector trait:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: otel-sidecar-collector-sample
+spec:
+  components:
+    - type: "k8s-objects"
+      name: "otel-deploy"
+      properties:
+        objects:
+          - apiVersion: apps/v1
+            kind: Deployment
+            metadata:
```

Comment:
Can we use the webservice component as an example instead? It's the most commonly used type of component.

Reply:
Done 👍🏽